apps/frontend/src/pages/project/project.tsx

```diff
@@ -31,34 +40,45 @@ const Project = (): JSX.Element => {
 
 	const isRejected = projectStatus === DataStatus.REJECTED;
 
-	const hasProject = Boolean(project);
+	const hasProject = project !== null;
 
 	if (isRejected) {
 		return <NotFound />;
 	}
 
+	if (!hasProject) {
+		return (
+			<PageLayout isLoading={isLoading}>
+				<></>
+			</PageLayout>
+		);
+	}
```

Comment:
looks not good, we need to either remove this condition at all if this case not possible, or use `!hasProject && !isLoading` to return nothing (without layout)
```suggestion
```

Reply:
Both variants did not work -- first because this component don't know that insides of PageLayout won't be rendered, while loading is finished and project loaded successfuly, and second -- because if `hasProject` is false, `isLoading` might be true that will prevent returning, so type sysyem cannot infer that our layout insides will be rendered only when project is not null.

Of course I can use `as ProjectGetAllItemResponse` to solve the issue with types (cause we surely now that here project cannot be null), but with this code looks a bit weird (there are 4 places, there project used). So I decided to bring back initial variant with those `{hasProject && ...` check and 2 extra tabulations for now.